lib/config/options/index.ts

```diff
@@ -705,7 +705,9 @@ const options: RenovateOptions[] = [
     name: 'autodiscoverFilter',
     description: 'Filter the list of autodiscovered repositories.',
     stage: 'global',
-    type: 'string',
+    type: 'array',
```

Comment:
Do we want a config migration for the change from a `string` to `array`?

Reply:
I think this is ok, as this is a breaking change and we cannot guarantee that we can migrate all config variations because of the massaging.     
But maybe the others have another opinion.

Reply:
No migrations feasible in this case, we need to document it